packages/server/computation_container/client/computation_to_db/join_table.cpp

```diff
@@ -383,16 +387,13 @@ std::string writeVJoinTable(
             new_row.emplace_back(r2[it]);
         }
 
-        new_table.emplace_back(new_row);
+        writer.emplace(new_row);
```

Comment:
It is essential.
Before: Save after putting it all together
After: Save one line at a time